ci-operator/config/openshift/operator-framework-tooling/openshift-operator-framework-tooling-main.yaml

```diff
@@ -0,0 +1,63 @@
+base_images:
+  commitchecker:
+    name: commitchecker
+    namespace: ci
+    tag: "4.14"
+  ocp_builder_rhel-8-golang-1.20-openshift-4.15:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.15
+  os:
+    name: centos
+    namespace: origin
+    tag: stream8
+binary_build_commands: GO111MODULE=on GOPROXY=https://proxy.golang.org make build
```

Comment:
Do we need the 2 env vars - aren't they defaults now?

Reply:
The CI somewhere sets the default to vendor so I did this to set it back